test/factories/users.rb

```diff
@@ -9,7 +9,7 @@
     trait :with_clients do
       client_amt = 5
       clients do |client|
-        1.step(client_amt).to_a.map { |n| client.association(:client) }
```

Comment:
Because I've discovered the much nicer and more obvious way to create the mappable array with `Array.new(amt)` doing the other factory. So it makes sense to change it everywhere it is used to the more reasonable way.